src/nidigital/examples/nidigital_burst_with_start_trigger.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/python
+
+import argparse
+import nidigital
+import os
+import sys
+
+
+def example(resource_name, options, trigger_source=None, trigger_edge=None):
+
+    with nidigital.Session(resource_name=resource_name, options=options) as session:
+
+        dir = os.path.join(os.path.dirname(__file__), 'burst_with_start_trigger_files')
+
+        # Load the pin map (.pinmap) created using the Digital Pattern Editor
+        pin_map_filename = os.path.join(dir, 'PinMap.pinmap')
+        session.load_pin_map(pin_map_filename)
+
+        # Load the specifications (.specs), levels (.digilevels), and timing (.digitiming) sheets created using the Digital Pattern Editor
+        spec_filename = os.path.join(dir, 'Specifications.specs')
+        levels_filename = os.path.join(dir, 'PinLevels.digilevels')
+        timing_filename = os.path.join(dir, 'Timing.digitiming')
+        session.load_specifications_levels_and_timing(spec_filename, levels_filename, timing_filename)
+
+        # Apply the settings from the levels and timing sheets we just loaded to the session
+        session.apply_levels_and_timing(levels_filename, timing_filename)
+
+        # Loading the pattern file (.digipat) created using the Digital Pattern Editor
+        pattern_filename = os.path.join(dir, 'Pattern.digipat')
+        session.load_pattern(pattern_filename)
+
+        if trigger_source is not None:
+            # Specify a source and edge for the external start trigger
+            session.start_trigger_type = nidigital.TriggerType.DIGITAL_EDGE
+            session.digital_edge_start_trigger_source = trigger_source
+            session.digital_edge_start_trigger_edge = nidigital.DigitalEdge.RISING if trigger_edge == 'Rising' else nidigital.DigitalEdge.FALLING
+
+        # If start trigger is configured, waiting for the trigger to start bursting and then blocks until the pattern is done bursting
+        # Else just start bursting and block until the pattern is done bursting
+        session.burst_pattern(start_label='new_pattern')
+        if trigger_source is None:
+            print('Start bursting pattern')
+        else:
+            print('Wait for start trigger and then start bursting pattern')
+
+        # Disconnect all channels using programmable onboard switching
+        session.selected_function = nidigital.SelectedFunction.DISCONNECT
+    print('Done bursting pattern')
+
+
+def _main(argsv):
+    parser = argparse.ArgumentParser(description='Demonstrates how to create and configure an instrument session and to burst a pattern on the digital pattern instrument using a start trigger.', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument('-n', '--resource-name', default='PXI1Slot2,PXI1Slot3', help='Resource name of a NI digital pattern instrument. Ensure the resource name matches the instrument name in the pinmap file.')
```

Comment:
why default to multi-instrument? just to show it's possible?

Reply:
@sbethur requested that the Python examples show usage multiple instruments to show off channel expansion just like their counterparts in LV and .NET.